liqd_product/templates/header.html

```diff
@@ -5,7 +5,7 @@
     <div class="l-wrapper">
         <div class="platform-brand main-header__brand">
             {% spaceless %}
-                <a href="{% if PARTNER %}{% url 'partner' partner_slug=PARTNER.slug %}{% else %}/{% endif %}" class="platform-brand__link">Beteiligung.in{% if PARTNER %}/<span class="marked">{{ PARTNER.name }}</span>{% endif %}</a>
+                <a href="{% if PARTNER %}{% url 'partner' partner_slug=PARTNER.slug %}{% else %}/{% endif %}" class="platform-brand__link">{% if PARTNER %}<span class="marked">{{ PARTNER.name }}</span>{% else %}Beteiligung.in{% endif %}</a>
```

Comment:
Now there's nothing on the landing page (and currently in the dashboard as well). Do we also want something there?

Reply:
I don't think i know what you mean? If there's no partner it displays Beteiligung.in and goes to the bet.in homepage, I thought the header in the dashboard is the other story and this one was just for the partner landing page?

Reply:
OMG! I can't read code anymore!